test/e2e/runner/setup.go

```diff
@@ -500,3 +516,78 @@ func UpdateConfigStateSync(node *e2e.Node, height int64, hash []byte) error {
 	bz = regexp.MustCompile(`(?m)^trust_hash =.*`).ReplaceAll(bz, []byte(fmt.Sprintf(`trust_hash = "%X"`, hash)))
 	return os.WriteFile(cfgPath, bz, 0o644) //nolint:gosec
 }
+
+// tcCommands generates a list of tc (traffic control) commands to emulate
```

Comment:
So this guy creates the `emulate-latency.sh` script?

Reply:
Do you mind moving this function to a different file, like `latency.go` or `latency_emulation.go`?

Reply:
`tcCommands` generates the content of `emulate-latency.sh`. I moved this function to to a new file `latency_emulation.go`.